cloudify_cli/main.py

```diff
@@ -48,7 +50,10 @@
 from .commands import certificates
 from .commands import node_instances
 from .commands import maintenance_mode
-from .commands import audit_log
+if PY2:
+    from .commands import audit_log
+else:
+    from .async_commands import audit_log
```

Comment:
Is there a reason you didn't put the async form into the existing commands.audit_log and gate behaviour on available features?

Reply:
py2 must not even attempt to import a module with those functions, because being `async def`, using `async for`, they'll be a syntaxerror

Reply:
There's only one function in there (and not the command itself) that uses those keywords. Farming them out to a separate module and importing that only if $correct_py_ver would result in less repitition on defining the commands, etc.